data/compliance_profiles/CentOS/7/disa_stig.json

```diff
@@ -2536,6 +2536,12 @@
         ],
         "notes": "This will probably change in the future to a class inclusion method that is more clear"
       },
+      "svckill::mode": {
+        "identifiers": [
+          "CCI-000381"
```

Comment:
Why does this entry not have SRG-OS-000095-GPOS-00049 and RHEL-07-020000 identifiers

Reply:
There was no *specific* SRG that this tied to, just the overarching CCI that maps to multiple SRG entries.